# HG changeset patch
# User Rik <rik@octave.org>
# Date 1378784280 25200
#      Mon Sep 09 20:38:00 2013 -0700
# Node ID 3feff4d089538621b11dfc99aed1a29123b9dd28
# Parent  a1fa416c7c4aec22062aa0e5d7f1caffb194bf2d
dialog.cc: Rename local variables to avoid shadowed variable warning

* libgui/src/dialog.cc(FileDialog, accept): Rename 'filters' to 'name_filters'.

diff --git a/libgui/src/dialog.cc b/libgui/src/dialog.cc
--- a/libgui/src/dialog.cc
+++ b/libgui/src/dialog.cc
@@ -415,17 +415,17 @@ InputDialog::buttonCancel_clicked (void)
 
   
 void
 InputDialog::reject (void)
 {
   buttonCancel_clicked ();
 }
 
-FileDialog::FileDialog (const QStringList& filters, const QString& title,
+FileDialog::FileDialog (const QStringList& name_filters, const QString& title,
                         const QString& filename, const QString& dirname,
                         const QString& multimode)
   : QFileDialog()
 {
   // Create a NonModal message.
   setWindowModality (Qt::NonModal);
 
   setWindowTitle (title.isEmpty () ? " " : title);
@@ -451,17 +451,17 @@ FileDialog::FileDialog (const QStringLis
       setAcceptMode (QFileDialog::AcceptOpen);
     }
   else                           // uigetfile multiselect=off
     {
       setFileMode (QFileDialog::ExistingFile);
       setAcceptMode (QFileDialog::AcceptOpen);
     }
 
-  setNameFilters (filters);
+  setNameFilters (name_filters);
 
   selectFile (filename);
   
   connect (this,
            SIGNAL (finish_input (const QStringList&, const QString&, int)),
            &uiwidget_creator,
            SLOT (filedialog_finished (const QStringList&, const QString&,
                                       int)));
@@ -488,16 +488,16 @@ void FileDialog::accept(void)
   // pull path names, so fix it.
 
   for (int i = 0; i < string_result.size (); i++)
     string_result[i] = QFileInfo (string_result[i]).fileName ();
 
 
   path = directory ().absolutePath ();
 
-  QStringList filters = nameFilters ();
-  idx = filters.indexOf (selectedNameFilter ()) + 1;
+  QStringList name_filters = nameFilters ();
+  idx = name_filters.indexOf (selectedNameFilter ()) + 1;
   
   // send the selected info
   emit finish_input (string_result, path, idx);
   done (QDialog::Accepted);
 }
 
